librz/bp/bp.c

```diff
@@ -330,15 +363,15 @@ RZ_API int rz_bp_del_index(RzBreakpoint *bp, int idx) {
 	return false;
 }
 
-RZ_API int rz_bp_size(RzBreakpoint *bp) {
+RZ_API int rz_bp_size(RzBreakpoint *bp, int bits) {
```

Comment:
Doxygen.

Reply:
also RZ_NONNULL, and asserts if they are not there